packages/twenty-server/src/engine/metadata-modules/workspace-migration/workspace-migration.entity.ts

```diff
@@ -28,8 +28,8 @@ export interface WorkspaceMigrationColumnDefinition {
   columnType: string;
   enum?: WorkspaceMigrationEnum[];
   isArray?: boolean;
-  isNullable?: boolean;
```

Comment:
making everything required as it's great to have this information available while running the migration. I prefer a simple developer experience where all the info is present rather than asking ourselves if isNullable or defaultValue is present in this case.
In practice it's almost always filled (I have fixed the places where it was not the case)

Reply:
@Weiko FYI for fast follow next week